src/renderer/helpers/api/local.js

```diff
@@ -19,6 +19,33 @@ const TRACKING_PARAM_NAMES = [
   'utm_content',
 ]
 
+function getRandomIosVersion() {
+  const iosVersions = [
+    '17.5.1',
+    '17.5',
+    '17.4.1',
+    '17.4',
+    '17.3.1',
+    '17.3',
+  ]
+
+  return iosVersions[Math.floor(Math.random() * iosVersions.length)]
+}
+
+function getRandomYouTubeIosClientVersion() {
+  const versions = [
+    '19.29.1',
+    '19.28.1',
+    '19.26.5',
+    '19.25.4',
+    '19.25.3',
+    '19.24.3',
+    '19.24.2',
+  ]
```

Comment:
Can the constant arrays be put outside to avoid allocations each time the function is called?

Reply:
Done :)